kubectl-directcsi/util/util.go

```diff
@@ -438,7 +438,6 @@ func CreateDaemonSet(ctx context.Context, kClient clientset.Interface, name, ide
 			newHostPathVolume(volumeNameCSIRootDir, csiRootPath),
 			newHostPathVolume(volumeNameDevDir, volumePathDevDir),
 			newHostPathVolume(volumeNameSysDir, volumePathSysDir),
-			newHostPathVolume(volumeNameProcDir, volumePathProcDir),
```

Comment:
Can we remove these constants - `volumeNameProcDir`, `volumePathProcDir` as we are not using it anywhere else

Reply:
done